src/interfaces/nav-controller-interface.ts

```diff
@@ -0,0 +1,31 @@
+import { LitElement } from 'lit-element';
+import { MetadataResponse } from '@internetarchive/search-service';
+import { ModalManager } from '@internetarchive/modal-manager';
+import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
+import { IntMenuShortcut } from './menu-interfaces';
+
+export interface IntNavController extends LitElement {
```

Comment:
What is this interface getting used for? I only see `BookNavigator` inheriting from it below.